java/backup-restore/src/main/java/com/instaclustr/cassandra/backup/cli/RestoreApplication.java

```diff
@@ -48,5 +48,9 @@ public void run() {
         final Operation operation = operationsService.submitOperationRequest(request);
 
         await().forever().until(() -> operation.state.isTerminalState());
+
+        if (operation.state == Operation.State.FAILED) {
+            throw new IllegalStateException("Restore operation was not successful.");
```

Comment:
this makes command return with exit code 1 so we can detect restore has failed in init container so whole init container will fail

Reply:
:+1: